### DISCUSSION TITLE
Bitcoin provider Hardware wallet bug

### DISCUSSION THREAD

**Discussion by 0xboga** [phantom/docs#254] [Help & Support]
### Summary

Hello, I'm experiencing an issue with Bitcoin integration in my DApp using Phantom. Signing simple messages works fine, but when trying to sign a PSBT with a hardwallet, I encounter the following error:

`[btc provider][internalError] Internal error.`

Here's the stack trace for more details:

`Error: [btc provider][internalError] Internal error. at Dn.<anonymous> (<anonymous>:3:125128) at [Generator.next](http://generator.next/) (<anonymous>) at f (<anonymous>:1:1100)`

The PSBT I'm trying to sign is:
`cHNidP8BAF4CAAAAATuh02Sptfv6KhXInDeAkG7eF/OY54tmu3e2RAlY/uFvAAAAAAD/////AQDh9QUAAAAAIlEgW4PzvJGOTkXXq+ykRcTJi/KO4kpJgclx33GVPOqBMw0AAAAAAAEA/XgCAgAAAAABAyGbEIbGijWoA7J2LOzOuoQTtHGE9w+s7H6S2vnym3fTAQAAABcWABTTQ/7iGRFUeITYLzDvbGPr6RlElf////+pXx51AeIU2ENu/ZPKcOUMTAaCScxauviJt7QT0J7xewMAAAAXFgAU00P+4hkRVHiE2C8w72xj6+kZRJX/////xkkFauqFYMh0tZXqa1Yt2DQX6DetdQLgacvngg27/zUCAAAAFxYAFNND/uIZEVR4hNgvMO9sY+vpGUSV/////wPoAwAAAAAAACJRIFuD87yRjk5F16vspEXEyYvyjuJKSYHJcd9xlTzqgTMNsAMAAAAAAAAXqRT6AchAvLdKxWd9w3izi78jwNCkiYeYOgAAAAAAABepFMThF41tyEJuvZzX7mTzGBtqA7rBhwJHMEQCIGo6bSpGyQ5yg4Ie2iLWEPgmT5YxPVyzGaTIYNCU7wwQAiAZ67AgwHNhjASmuMvw1NCmoF/+m/1G134XZo9GGLAf0wEhA4qdm9EfeR1fNGY59LiFwMPsfPqnElsDr6V5ooene2AxAkcwRAIgdEEHzLWwjh8j+PWHU0XkNSUM28BdaiSummIdD9+Of9kCIF7+NYx66SOYT+YJZ02oa529GF0jrDkHU9pqRSetPisIASEDip2b0R95HV80Zjn0uIXAw+x8+qcSWwOvpXmih6d7YDECRzBEAiAuSMWE5OSnGrsobAEkWUWovWUuYNGwOUTGVsLID3XHMQIgWmusOms24sIx3MGTWFnLT6KNnGqLDjO3BosywV4L3bQBIQOKnZvRH3kdXzRmOfS4hcDD7Hz6pxJbA6+leaKHp3tgMQAAAAABASvoAwAAAAAAACJRIFuD87yRjk5F16vspEXEyYvyjuJKSYHJcd9xlTzqgTMNAQMEgwAAAAEXIM/CN4wdki2l1R2FlzkcazGi6RJ3gz8IJ+Eq20Q1rchFAAA=`

I've tried to test the HD Wallet integration on multiple websites that use Bitcoin like MagicEden and it seems that for them its not working as well. 

Thank you so much and pls dm me if u have any further questions. Thanks.

### Steps to Reproduce

1. Generate a PSBT
2. Connect in Phantom wallet extension using Ledger
3. Try to sign the PSBT

### Phantom Version

24.10.2

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @0xboga,

My apologies for the delay in response. Would you mind confirming this issues is still persistent, please? I will investigate ASAP if so.

**Reply by 0xboga** [reply]
Hello @adamdelphantom,

Yes, the problem still persists as described above. Additional information, including logs and screen recordings, were sent with support ticket number 86846.

Thank you for your assistance.

**Reply by adamdelphantom** [reply]
@0xboga Thanks for the confirmation! We are investigating.
